### PR TITLE
Fix quantization tutorials (imports, syntax, and style)

### PR DESCRIPTION
**Pull request by andrewor14**
Summary: This commit fixes the quantization tutorials such that
they can be run smoothly by the user.

Test Plan: Ran the updated tutorials without problem.

Reviewers: jerryzh168

Subscribers: jerryzh168, supriyar

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]

File: prototype_source/fx_graph_mode_ptq_static.rst [hunk 1]
```diff
@@ -294,6 +291,7 @@ Utility functions related to ``qconfig`` can be found in the `qconfig <https://g
   
 .. code:: python  
   
+    from torch.ao.quantization.quantize_fx import prepare_fx
```
Comment (jerryzh168) [inline prototype_source/fx_graph_mode_ptq_static.rst:294]: should we move imports to the beginning?

File: prototype_source/fx_graph_mode_ptq_static.rst [hunk 2]
```diff
@@ -75,7 +75,7 @@ Download the `torchvision resnet18 model <https://github.com/pytorch/vision/blob
     import sys  
     import torch.quantization
```
Comment (jerryzh168) [inline prototype_source/fx_graph_mode_ptq_static.rst:76]: we should probably remove torch.quantization now

Comment (jerryzh168) [reply prototype_source/fx_graph_mode_ptq_static.rst:76]: also in L95 of this file:
```
from torch.quantization import get_default_qconfig, quantize_jit
```
we should remove quantize_jit

File: prototype_source/fx_graph_mode_ptq_static.rst [hunk 3]
```diff
@@ -75,7 +75,7 @@ Download the `torchvision resnet18 model <https://github.com/pytorch/vision/blob
     import sys  
     import torch.quantization   
 
-    # Setup warnings    
+    # Set up warnings
     import warnings 
     warnings.filterwarnings(    
         action='ignore',
```
Comment (jerryzh168) [inline prototype_source/fx_graph_mode_ptq_static.rst:81]: please also update all `torch.quantization` in this file to `torch.ao.quantization`, another example is in L87

File: advanced_source/dynamic_quantization_tutorial.py [hunk 4]
```diff
@@ -98,9 +98,9 @@ def __len__(self):
 class Corpus(object):
     def __init__(self, path):
         self.dictionary = Dictionary()
-        self.train = self.tokenize(os.path.join(path, 'train.txt'))
-        self.valid = self.tokenize(os.path.join(path, 'valid.txt'))
-        self.test = self.tokenize(os.path.join(path, 'test.txt'))
+        self.train = self.tokenize(os.path.join(path, 'wiki.train.token'))
+        self.valid = self.tokenize(os.path.join(path, 'wiki.valid.token'))
+        self.test = self.tokenize(os.path.join(path, 'wiki.test.token'))
```
Comment (malfet) [inline advanced_source/dynamic_quantization_tutorial.py:103]: Why are you doing those changes? Where those files are supposed to be coming from?

**Review by malfet — CHANGES_REQUESTED** [review]
This breaks CI, namely because it references non-existing data files

File: advanced_source/dynamic_quantization_tutorial.py (hunk 4 above)
Comment (andrewor14) [reply advanced_source/dynamic_quantization_tutorial.py:103]: These were the paths when I downloaded the dataset myself. Without these changes I was not able to run the tutorials. I guess CI downloads the files differently somehow

Comment (andrewor14) [reply advanced_source/dynamic_quantization_tutorial.py:103]: To unblock this PR maybe we should just revert these changes and investigate separately. How does that sound @jcaip?

Comment (malfet) [reply advanced_source/dynamic_quantization_tutorial.py:103]: Hmm, downloaded from where? For tutorials build, download locations are defined in (plan to move them to json file, but this is where they are for now)
https://github.com/pytorch/tutorials/blob/4918fe4bf3df96b233ca134c985ffb09ea45f92c/Makefile#L93-L94

Comment (jcaip) [reply advanced_source/dynamic_quantization_tutorial.py:103]: I was using a different wikitext version which has changed the naming convention of those files, which is causing the error. I will update the internal tutorial guide and revert the filepaths, which I think should fix everything. Thanks for the help @malfet!

**Pull request merged**